### PR TITLE
raise hand feature - stateful client

### PR DESCRIPTION
**Pull request by rzdor**
What

I've added implementation for Raise/Lower Hand button to components.
Updated composites and Calling test app to use new element.
Added states and subscribers to hand updates for new published/removed states.

image
image

Why
Implementation for SDK raise hand feature

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Comment by rzdor** [comment]
breaking-change

**Comment by rzdor** [comment]
#breaking-change

File: change-beta/@azure-communication-react-da2d5b43-20a8-4bc0-8cbe-09ceb376a06f.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "move client commits to separate branch",
```
Comment (alkwa-msft) [inline change-beta/@azure-communication-react-da2d5b43-20a8-4bc0-8cbe-09ceb376a06f.json:3]: What is this change really doing. I dont think this message constructively adds any value to this commit. Perhaps it should be "adding raised hand functionality to stateful client"

File: change/@azure-communication-react-da2d5b43-20a8-4bc0-8cbe-09ceb376a06f.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "move client commits to separate branch",
```
Comment (alkwa-msft) [inline change/@azure-communication-react-da2d5b43-20a8-4bc0-8cbe-09ceb376a06f.json:3]: same comment about this changelog message

File: packages/calling-stateful-client/review/beta/calling-stateful-client.api.md [hunk 3]
```diff
@@ -251,6 +253,13 @@ export interface OptimalVideoCountFeatureState {
     maxRemoteVideoStreams: number;
 }
 
+// @public
+export interface RaiseHandCallFeature {
+    raisedHands: RaisedHand[];
+    // (undocumented)
+    userRaisedHand?: RaisedHand;
```
Comment (alkwa-msft) [inline packages/calling-stateful-client/review/beta/calling-stateful-client.api.md:260]: I dont like this name. can we change this to "myRaisedHand" or something? or localRaisedHand? Can we bring this up in the feature weekly?

File: packages/calling-stateful-client/src/RaiseHandSubscriber.ts [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  private _callIdRef: CallIdRef;
+  private _context: CallContext;
+  private _raiseHand: RaiseHandCallFeature;
+
+  constructor(callIdRef: CallIdRef, context: CallContext, raiseHand: RaiseHandCallFeature) {
+    this._callIdRef = callIdRef;
+    this._context = context;
+    this._raiseHand = raiseHand;
+
+    this.subscribe();
```
Comment (alkwa-msft) [inline packages/calling-stateful-client/src/RaiseHandSubscriber.ts:26]: if all of the constructors call subscribe then this should be fine. The problem exists where people assume the constructor is not subscribing which can cause unintended issues.

**Review by alkwa-msft — APPROVED** [review]

File: change-beta/@azure-communication-react-da2d5b43-20a8-4bc0-8cbe-09ceb376a06f.json (hunk 1 above)
Comment (dmceachernmsft) [reply change-beta/@azure-communication-react-da2d5b43-20a8-4bc0-8cbe-09ceb376a06f.json:3]: message type should also be pre-release

File: change-beta/@azure-communication-react-da2d5b43-20a8-4bc0-8cbe-09ceb376a06f.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```
Comment (dmceachernmsft) [inline change-beta/@azure-communication-react-da2d5b43-20a8-4bc0-8cbe-09ceb376a06f.json:2]: Make sure that there is the right number of changelogs, you should be able to fix this by running `rush changelog` again 😊

**Review by dmceachernmsft — APPROVED** [review]

**Pull request merged**